src/app/sentry/init.ts

```diff
@@ -0,0 +1,26 @@
+import * as Sentry from "@sentry/browser"
+
+type Environment = "production" | "acceptance"
+
+const getEnvironment = (): Environment | undefined => {
+  const { hostname } = window.location
+  const environment =
+    hostname === "zaken.amsterdam.nl" ? "production" :
+    hostname === "acc.zaken.amsterdam.nl" ? "acceptance" :
+    undefined
+  return environment
+}
+
+export default () => {
+  // TODO Get environment from .env file
+  const environment = getEnvironment()
+  if (environment === undefined) return
```

Comment:
Not really a big fan of this.
Just don't init if there is no SENTRY_DSN set.

Reply:
The environment is needed to differentiate between `acceptance` and `production` anyway. This is exactly what could be set using different Docker builds between environments. Otherwise a runtime check is necessary.